src/config.py

```diff
@@ -537,11 +542,20 @@ def server_properties(self) -> List[str]:
 
         return properties
 
+    @staticmethod
+    def _translate_config_key(key):
+        """Format config names into server properties, blacklisted property are commented out.
+
+        Returns:
+            String with Kafka configuration name to be placed in the server.properties file
+        """
+        return key.replace("_", ".") if key not in SERVER_PROPERTIES_BLACKLIST else f"# {key}"
+
     @property
     def config_properties(self) -> List[str]:
         """Configure server properties from config."""
         return [
-            f"{conf_key.replace('_', '.')}={str(value)}"
+            f"{self._translate_config_key(conf_key)}={str(value)}"
```

Comment:
Good catch! Were we adding everything before? I'm surprised Kafka didn't complain :upside_down_face:

Reply:
I suppose Kafka ignored properties that it does not recognize. However, with this small modification is a bit clearer which properties belongs to Kafka and which one do not. This modification was a bit more important for Zookeeper, that is somewhat stricter with property keys